### PR TITLE
registry: MCP mainnet deploy

### PR DESCRIPTION
**Pull request by tynes**
**Description**

Deploys the MCP contracts to mainnet. Verifies on etherscan.
Implementation addresses are included so the canonical deployments
are tracked. In the future, we can move away from adding values to
network specific files if a standard create2 flow is adopted for
all chains. A future release can work towards this.

```bash
git checkout op-contracts/v1.3.0-rc.1
forge clean

export IMPL_SALT="op/acc"

forge script scripts/Deploy.s.sol \
  --sig 'deployImplementations()' \
  --rpc-url $ETH_RPC_URL \
  --private-key $PRIVATE_KEY \
  --broadcast \
  --verify
```

### CODE REVIEW

File: superchain/implementations/networks/mainnet.yaml
Comment (mds1) [inline superchain/implementations/networks/mainnet.yaml:9]: Should we delete the 1.6.0 version? It has immutables so it's not an MCP version, so I don't think we want that address here since it's OP Mainnet specific

Comment (mds1) [reply superchain/implementations/networks/mainnet.yaml:9]: I see in https://github.com/ethereum-optimism/superchain-registry/blob/main/CONTRIBUTING.md#implementations that it's supposed to be a canonical set of implementation deployments. I don't know what exactly that means pre-MCP, but it does make sense now, so I think we should delete 1.6.0 unless there is a purpose/use case for it I'm unaware of

**Comment by tynes** [comment]
Pushed [a51070d](https://github.com/ethereum-optimism/superchain-registry/pull/114/commits/a51070dc140552e046e592b0c55fc15fd171827f) which removes the non mcp contract address from the registry, good catch

File: superchain/implementations/networks/mainnet.yaml
Comment (mds1) [reply superchain/implementations/networks/mainnet.yaml]: Resolved by @tynes in a51070dc140552e046e592b0c55fc15fd171827f

**Review by mds1 — APPROVED** [review]

**Comment by tynes** [comment]
I cannot approve my own PR, wondering if i can get a review from @ethereum-optimism/go-reviewers

**Pull request merged**
